test/dotnet-new3.UnitTests/Approvals/DotnetNewHelp.CanShowHelp_Install.approved.txt

```diff
@@ -0,0 +1,15 @@
+﻿Description:
+  Installs a template package.
+
+Usage:
+  new3 install <package>... [options]
```

Comment:
does `...` in this usage imply repetition?

Reply:
yes, it means 1 or more argument.